README.md

```diff
@@ -3,10 +3,18 @@
 # Microverse Capstone
 
 # Description the project.
-For the first milestone in the process of creating my portfolio website, I have:
+For the fifth milestone in the process of my capstone project, I have:
 
-   - set up a new repository and prepare it for development using best practices .
-   - build the first 2 sections of the mobile website using the template 4 of the Microverse portfolios.
+    
+    - Implement the section "Featured speakers" with dynamic HTML.
+    - Deploy the project and check to make sure everything works as planned.
+
+# Acknowledgements
+   - Thanks to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
+
+# Video Presentation
```

Comment:
- [ ] [OPTIONAL] In the future I would recommend installing Krisp it gives your every week to record clear sound and removes the background noise, it works very well with zoom and loom.